specification/batchai/resource-manager/Microsoft.BatchAI/stable/2018-05-01/BatchAI.json

```diff
@@ -941,112 +941,6 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.BatchAI/workspaces/{workspaceName}/fileServers/{fileServerName}": {
```

Comment:
>fileServers [](http://example.com/codeflow?start=127&length=11)

Please follow the deprecation process @ [https://armwiki.azurewebsites.net/api_contracts/APIDeprecationPolicy.html](https://armwiki.azurewebsites.net/api_contracts/APIDeprecationPolicy.html)

If it's already followed please share the evidences for the same for approving.